pkg/resource/cache_subnet_group/sdk.go

```diff
@@ -150,6 +150,15 @@ func (rm *resourceManager) sdkFind(
 	if err != nil {
 		return nil, err
 	}
+
+	if ko.Status.Subnets != nil {
+		for _, subnetIdIter := range ko.Status.Subnets {
+			if subnetIdIter.SubnetIdentifier != nil {
+				ko.Spec.SubnetIDs = append(ko.Spec.SubnetIDs, subnetIdIter.SubnetIdentifier)
+			}
+		}
+	}
```

Comment:
nit: `range` and `append` work fine with nil arrays - so no need for the nil check

Reply:
Thanks! Actioned.